### PR TITLE
 Fix: Resolving Servo Telemetry Clashes and making DroneCAN Servo compatible.

### PR DESCRIPTION
**Pull request by loki077**
Problem Statement:
In Ottano, the use of DroneCAN Servo causes telemetry log clashes due to indexing messages by Actuator ID. This occurs because multiple servos (aileron, elevator, rudder) share the same Actuator ID, resulting in overlapping logs that cannot distinguish between individual servo outputs.

Solution:
Split Servo Outputs For each Servo:
To address this, each servo output is now uniquely defined by function and position (right/left), ensuring no telemetry clash:

```
SERVO6_FUNCTION,19  # Elevator Right  OUT1 56  
SERVO10_FUNCTION,4  # Aileron Right   OUT1 60  
SERVO11_FUNCTION,21 # Rudder Right    OUT2 61  

SERVO12_FUNCTION,19 # Elevator Left   OUT1 62  
SERVO13_FUNCTION,4  # Aileron Left    OUT1 63  
SERVO15_FUNCTION,21 # Rudder Left     OUT1 64
```

Broadcast Servo Outputs Across Both Buses:
The changes ensure all six servo outputs are broadcasted on both CAN buses, facilitating flexibility to relocate a servo between buses in the future. As in Volanti Right Wing and Left Tail are on one bus and in Ottano Right Wing and Right Tail are on same bus. Also, the impact on BUS bandwidth is not massive.

Scope and Implementation:
This change is specifically needed for Ottano and the upcoming Octtano platforms.
It has been ported to Volanti for future compatibility without impacting current aircraft functionality. This prepares Volanti for potential DroneCAN servo integrations.

Testing Requirements:
Testing is required on IronBird first, followed by validation on Ottano and Volanti.

Upgrade Impact:
When upgrading Ottano or Volanti, users must reset the Left Wing/Tail Parameters (CPN) to default after writing the updated firmware.

### CODE REVIEW

**Comment by loki077** [comment]
I need a review from @LachlanConn @robertlong13 and @Jono453 on this.

File: libraries/AP_HAL_ChibiOS/hwdef/CarbonixCommon/cpn_params/Volanti/Volanti-LTail.parm
Comment (robertlong13) [inline libraries/AP_HAL_ChibiOS/hwdef/CarbonixCommon/cpn_params/Volanti/Volanti-LTail.parm:1]: Let's remove the CPN changes from consideration here. We can keep the Cube changes.

Defaulting CPN params on an update is a pain that I'd like to avoid, and it buys us nothing. The Cube changes on their own pave the way for DroneCAN servos in Volanti, and no living Ottanos have servo CPNs anymore.

File: libraries/AP_HAL_ChibiOS/hwdef/CarbonixCommon/defaults.parm
Comment (robertlong13) [inline libraries/AP_HAL_ChibiOS/hwdef/CarbonixCommon/defaults.parm]: Shouldn't this be 24352?

**Review by robertlong13 — CHANGES_REQUESTED** [review]

File: libraries/AP_HAL_ChibiOS/hwdef/CarbonixCommon/cpn_params/Volanti/Volanti-LTail.parm
Comment (loki077) [reply libraries/AP_HAL_ChibiOS/hwdef/CarbonixCommon/cpn_params/Volanti/Volanti-LTail.parm:1]: Agreed, I was about to mention that point where I will remove the Volanti CPN changes.

File: libraries/AP_HAL_ChibiOS/hwdef/CarbonixCommon/defaults.parm
Comment (loki077) [reply libraries/AP_HAL_ChibiOS/hwdef/CarbonixCommon/defaults.parm]: yes I forgot to update this but it should be 16160
image

**Review by Jono453 — COMMENTED** [review]
Legacy Ottano CPNs do not need the params changed as they have been replaced by CPCs. Instead, the parameters in each of the 6 x droneCAN servos will need to be updated to have their actuation_id match  their new CUBE servo mappings

File: libraries/AP_HAL_ChibiOS/hwdef/CarbonixCommon/cpn_params/Volanti/Volanti-LTail.parm
Comment (loki077) [reply libraries/AP_HAL_ChibiOS/hwdef/CarbonixCommon/cpn_params/Volanti/Volanti-LTail.parm:1]: changes done

File: libraries/AP_HAL_ChibiOS/hwdef/CarbonixCommon/defaults.parm
Comment (robertlong13) [reply libraries/AP_HAL_ChibiOS/hwdef/CarbonixCommon/defaults.parm]: Need Servo 15 checked, not Servo 14

**Comment by LachlanConn** [comment]
Looks good, once the CPN parts are removed I'll approve.

**Review by robertlong13 — APPROVED** [review]

**Review by Jono453 — APPROVED** [review]
Ill update the dronecan servo QC page once this is ready as well

**Review by LachlanConn — APPROVED** [review]

**Review by Pradeep-Carbonix — APPROVED** [review]
LGTM

**Comment by loki077** [comment]
going to be tested by https://carbonix.atlassian.net/browse/FTE-22

**Pull request merged**
